src/iss.js

```diff
@@ -128,9 +129,13 @@ export function mungeUrlQuery(url_: string, data: Object): string {
     return url_ + joiner + serialized;
 }
 
-export async function request(path: string, data: ?searchRequest): Object {
+export async function request(
+    path: string,
+    data: ?searchRequest
+): Promise<Object> {
```

Comment:
I think now `async` functions no longer need to return `Promise<blah>` and can just return `blah`.